### ISSUE TITLE
Accessibility axe test fails because of missing aria attribute at the input

### ISSUE THREAD

**Issue by leonp5** [daemons88/ngx-custom-material-file-input#2]
As in the title said:
The Jest test with axe show the following error:

```
Error: expect(received).toHaveNoViolations(expected)

Expected the HTML found at $('input[type="file"]') to have no violations:

<input _ngcontent-a-c3282449856="" type="file" accept=".png,.jpg,.jpeg">

Received:

"Form elements must have labels (label)"

Fix any of the following:
  Form element does not have an implicit (wrapped) <label>
  Form element does not have an explicit <label>
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element has no placeholder attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.9/label?application=axeAPI
```
The problem seems to be the input element which is generated by `ngx-custom-material-file-input`. There isn't any accessibility attribute on it. Is there any solution to pass the aria-attribute to the input? Or could any or multiple of the suggestions of the axe proposals be implemented?

![Image](https://github.com/user-attachments/assets/dc042b3a-f8a8-431c-91fc-87f53a53c641)

**Comment by daemons88** [comment]
Thanks for reporting it!

Give me this week, I'll try to upload a new version on the weekend.

I'll add a default aria attribute and check if is not too complicated to add the option to pass as a param the aria attribute.

Which version of angular are you using?

**Comment by daemons88** [comment]
Fixed in the latest version:
https://github.com/daemons88/ngx-custom-material-file-input/releases/tag/19.0.1

**Comment by leonp5** [comment]
Thanks for fixing it so fast 🙂
